lib/lamian/sidekiq_raven_middleware.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# A sidekiq middleware
+class Lamian::SidekiqRavenMiddleware
+  # Adds current lamian log to the extra part of raven events generated inside sidekiq jobs
+  def call(*)
+    Lamian.run do
+      begin
+        yield
+      rescue
```

Comment:
I think it should be `rescue Exception`. Exception class hierarchy in ruby is broken and i see no problem to catch all exceptions if we re-raise them